.env.example

```diff
@@ -0,0 +1,6 @@
+NODE_ENV=development
+PORT=3000
+STATIC_FILES_PATH=../public
+CORS_ORIGIN=http://localhost:5173
```

Comment:
Do you need CORS config? You need this config when you will deploy frontend and backend in different servers. Other wise, you can use vite proxy in frontend config

Reply:
i dont need it 👍, we will use proxy